src/main/java/Duke.java

```diff
@@ -1,10 +1,88 @@
+import java.util.*;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner input = new Scanner(System.in);
+        ArrayList<Task> currList = new ArrayList<>();
+
+        //Duke starts up with an introduction
+        System.out.println("Hey, I'm Duke!\n" + "How can I help you?");
+
+        while (input.hasNextLine()) {
+            String command = input.nextLine();
+            String[] commandArr = command.split(" ", 2);
+
+            if (command.equals("bye")) {
+                //Duke exits when user types 'bye' command
+                System.out.println("Aw. It was nice chatting with you! Don't forget me! :)");
+                break;
+            } else if (command.equals("list")) {
+                //Every item is listed out when user types 'list' command
+                System.out.println("Here are the tasks in your list:");
+                for (int i = 0; i < currList.size(); i++) {
+                    Task item = currList.get(i);
+                    System.out.println((i + 1) + "." + item.toString());
+                }
+            } else if ((commandArr[0]).equals("done")) {
+                //If user enters an invalid task number, prompt a message
+                if (Integer.parseInt(commandArr[1]) > currList.size()) {
+                    System.out.println("Did you miscount your tasks? "
+                            + "This task number isn't valid. Add it into my list!");
+                    continue;
+                }
+                //If valid, change status of task and output done task
+                Task item = currList.get(Integer.parseInt(commandArr[1]) - 1);
+                item.markAsDone();
+                System.out.println("Nice job! I've marked this task as done:\n"
+                        + item.toString());
+            } else if ((commandArr[0]).equals("delete")){
+                Task item = currList.remove(Integer.parseInt(commandArr[1]) - 1);
+                System.out.println("Gotcha. I've removed this task:\n"
+                        + item.toString()
+                        + "\nNow you have " + currList.size() + " task(s) in the list.");
+            } else {
+                String taskType = commandArr[0];
+                String wholeTask;
+                Task item;
+                switch (taskType) {
+                    case "todo":
```

Comment:
Do you think it will be more organised if the case blocks are indented at the same level as the switch block?